src/test/java/org/zalando/intellij/swagger/completion/CaretCompletionTest.java

```diff
@@ -0,0 +1,96 @@
+package org.zalando.intellij.swagger.completion;
+
+import com.intellij.util.net.HTTPMethod;
+import org.jetbrains.annotations.NotNull;
+import org.junit.*;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.zalando.intellij.swagger.completion.SwaggerFixture.JsonOrYaml;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+@RunWith(Parameterized.class)
+public class CaretCompletionTest {
+    @Rule
+    public TestName testName = new TestName();
+
+    private SwaggerFixture myFixture;
+    private final JsonOrYaml myJsonOrYaml;
+
+    @Parameterized.Parameters(name = "inputKind: {0}")
+    public static Object[] parameters() {
+        return JsonOrYaml.values();
+    }
+
+    public CaretCompletionTest(JsonOrYaml jsonOrYaml) {
+        myJsonOrYaml = jsonOrYaml;
+    }
+
+    @Before
+    public void setUpBefore() throws Exception {
+        myFixture = SwaggerFixture.forResourceFolder("testData/completion");
+    }
+
+    @After
+    public void tearDownAfter() throws Exception {
+        myFixture.tearDown();
+        myFixture = null;
```

Comment:
Sorry, don't understand. 
- call to tearDown is required. 
- setting field to null is not required but does not look wrong to me